composer.json

```diff
@@ -47,17 +49,17 @@
     "require-dev": {
         "dama/doctrine-test-bundle": "^6.7",
         "doctrine/doctrine-fixtures-bundle": "^3.0",
```

Comment:
```suggestion
        "doctrine/doctrine-fixtures-bundle": "^3.4.1",
```

Reply:
Thanks! I've updated this and all the other dependency versions.